staff/pep-delbano/pre-course/calculator/stylescalculadora.css

```diff
@@ -0,0 +1,94 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-size: 18px;
+  font-family: Helvetica,Arial,sans-serif;
+  background-color: /* Permalink - use to edit and share this gradient: http://colorzilla.com/gradient-editor/#f8ffe8+0,e3f5ab+41,b7df2d+100 */
+background: #f8ffe8; /* Old browsers */
+background: -moz-linear-gradient(top, #f8ffe8 0%, #e3f5ab 41%, #b7df2d 100%); /* FF3.6-15 */
+background: -webkit-linear-gradient(top, #f8ffe8 0%,#e3f5ab 41%,#b7df2d 100%); /* Chrome10-25,Safari5.1-6 */
+background: linear-gradient(to bottom, #f8ffe8 0%,#e3f5ab 41%,#b7df2d 100%); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
+filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#f8ffe8', endColorstr='#b7df2d',GradientType=0 ); /* IE6-9 */
+}
+
+.page {
+  width: 700px;
+}
+
+header {
+  width: 70%;
+  margin: auto; /*para centrar dcha izda*/
+  margin-top: 30px;
```

Comment:
- Another way to write it:
margin: 30px 0 80px 0;

with this we can modify everything in one line.